docs/content/common-subs/create-ks-core-kind-cluster.md

```diff
@@ -0,0 +1,36 @@
+<!--create-ks-core-kind-cluster-start-->
+create the ks-core kind cluster
+```shell
+export KUBECONFIG=~/.kube/config
+kind create cluster --name ks-core --config - <<EOF
```

Comment:
There is no need here to diverge from the pattern that is mainly used in this document, `KUBECONFIG=~/.kube/confg command`.